pages/Nix/Hyprland on Home Manager.md

```diff
@@ -126,3 +126,43 @@ wayland.windowManager.hyprland.plugins = [
 
 For examples on how to build hyprland plugins using nix see the
 [official plugins](https://github.com/hyprwm/hyprland-plugins).
+
+## Fixing problems with themes
+
+If your themes for mouse cursor, icons or windows don't load correctly, try setting them with `home-manager`. `home-manager` enables a bunch of compatibility options that should make the themes load in all situations.
```

Comment:
```suggestion
If your themes for mouse cursor, icons or windows don't load correctly, try setting them with `home.pointerCursor` and `gtk.theme`, which enable a bunch of compatibility options that should make the themes load in all situations.
```

Reply:
You marked this as resolved but didn't apply the suggestion.

Reply:
Sorry, i've commited your suggestion now. (a47cd03)

Please tell me if the other changes are ok or should be removed